src/OrbitBase/LoggingUtilsTest.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "LoggingUtils.h"
+#include "absl/strings/str_format.h"
+#include "absl/time/clock.h"
+
+namespace {
+std::filesystem::path CreateTestLogFilePath(const absl::Time& timestamp) {
+  std::filesystem::path test_log_dir = "C:/OrbitAppDataDir/logs";
```

Comment:
does it work on linux? Maybe use https://en.cppreference.com/w/cpp/filesystem/temp_directory_path

Reply:
The name of this function might be misleading. It generates a path used for testing and will not create a real directory in the file system. It is a good idea to use `temp_directory_path`, but  `temp_directory_path` returns an empty path on error and we still need to generate a test path in such a case.

Maybe I should rename this function as `GenerateTestLogFilePath` or `GetTestLogFilePath`?

Reply:
ok, I see..